src/theory/quantifiers/sygus/sygus_reconstruct.h

```diff
@@ -213,27 +216,28 @@ class SygusReconstruct : public expr::NotifyMatch
    * example, given:
    *
    * CandSols = {
-   *  mainOb -> {(+ z1 1)},
-   *  z1 -> {(* z2 x)},
-   *  z2 -> {2}
+   *   c_mainOb -> {(c_+ c_z1 c_1)},
```

Comment:
`c_mainOb` is a skolem here?  I think it would be clearer if we used k_mainOb instead (I tend to use `k` to represent skolems).  

This also distinguishes this from c_+, which is a sygus constructor here I guess?

Reply:
Replaced `c_mainOb` with `k0`, which is the main skolem in the algorithm. Also, updated comments to be consistent with `z`s (represent free vars) and `k`s (represent skolems).